storage/innobase/handler/ha_innodb.cc

```diff
@@ -17540,15 +17544,27 @@ ib_logf(
 	switch(level) {
 	case IB_LOG_LEVEL_INFO:
 		sql_print_information("InnoDB: %s", str);
+#ifdef HAVE_SYSTEMD
+		sd_notifyf(0, "STATUS=InnoDB: Info: %s", str);
+#endif
```

Comment:
* same comment as above about reducing the number of ifdef's and defining
  sd_notifyf to noop.
* oh, really? all innodb log messages are sent to systemd? I'd understand
  if you'd do it for startup, to show the startup progress. but *all* log
  messages for as long as mysqld runs?